source/user-guide/how-to-setup-code-scanning-locally.html.md.erb

```diff
@@ -0,0 +1,89 @@
+---
+owner_slack: "#modernisation-platform"
+title: How to setup code scanning software locally
+last_reviewed_on: 2024-09-02
+review_in: 6 months
+---
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-NXTCMQ7ZX6"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-NXTCMQ7ZX6');
+</script>
+
+# <%= current_page.data.title %>
+
+## Introduction
+
+At present we use checkov and trivy to do static code analsys this is done via github actions and use of a terraform module. 
+There has been a request to have the ability run these progams locally so that issue can be idenified before pushng a pull request up to the repository,
+this document will give direction on how to install the software and run them with the correct paramiters.
```

Comment:
parameters

Reply:
Currently, we use Checkov and Trivy for static code analysis, which is performed via GitHub Actions and use of a Terraform module. There has been a request to enable running these programs locally so that issues can be identified before pushing a pull request to the repository. This document provides instructions on how to install the software and run it with the correct parameters.

Reply:
resolved with latest commit